boot/zephyr/os.c

```diff
@@ -41,7 +41,7 @@
 #define CRYPTO_HEAP_SIZE 6144
 #else
 #  if !defined(MBEDTLS_RSA_NO_CRT)
-#  define CRYPTO_HEAP_SIZE 10240
+#  define CRYPTO_HEAP_SIZE 12032//11860
```

Comment:
This is kind of confusing, with the comment butted up like that. Maybe just have 12032?

Reply:
ops, post investigation artefact :)  commented value was the lowers seams to working.
I rounded it up to multiplication of 0x100.